app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java

```diff
@@ -31,15 +28,16 @@
 public class CameraViewActivity extends AppCompatActivity implements CameraBridgeViewBase.CvCameraViewListener2 {
```

Comment:
Perhaps add a short JavaDoc paragraph explaining the responsibilities of this activity.